Cauli/Configuration/Configuration.swift

```diff
@@ -25,6 +25,9 @@ import Foundation
 /// The Configuration is used to configure Cauli at initialization time.
 /// The `Configuration.standard` is a sensibly chosen configuration set.
 public struct Configuration {
+    /// A callback that returns a possibly modified `Record`
+    public typealias RecordModifier = ((Record) -> Record)
```

Comment:
Oh, we already have something like a `RecordModifier`. Have a look [here](https://github.com/cauliframework/cauli/blob/9fc3746aa6fbf3ae9fa6c81891eccf5fbee54974/Cauli/Florets/FindReplace/RecordModifier.swift) There are two differences.

1. It is in the `FindReplaceFloret` namespace. Since we are using it in this place as well, I would just remove it from that namespace.
2. It is a closure wrapped in a struct. The idea of that struct is to have easy factory functions on that struct.

What do you think about reusing that `RecordModifier`?

Reply:
Why did you split the RecordModifier up over three extensions?

Reply:
In general, I like to structure code via extensions. There is no more specific reason than that 😊

Reply:
I agree, but that looks like a step to far for me :-D
Do you have any recommendation where the `RecordModifier` should live, if I want to move it out of the `FindReplaceFloret`?

Reply:
I would be fine with just having one extension. One for the struct itself and one for the factory methods.

I would put it right in the frameworks namespace, so nothing special.